cmd/minikube/cmd/dashboard.go

```diff
@@ -79,6 +71,28 @@ var dashboardCmd = &cobra.Command{
 		if err != nil {
 			exit.WithError("Error getting client", err)
 		}
+
+		_, err = api.Load(pkg_config.GetMachineName())
+		if err != nil {
+			switch err := errors.Cause(err).(type) {
+			case mcnerror.ErrHostDoesNotExist:
+				console.OutStyle(console.Meh, "%q cluster does not exist", pkg_config.GetMachineName())
+				os.Exit(0)
```

Comment:
I think this should still be a non-zero exit. Perhaps exit.Unavailable?

Reply:
Thanks @tstromberg, done.